src/lightning_lite/plugins/__init__.py

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from lightning_lite.plugins.environments.cluster_environment import ClusterEnvironment
-from lightning_lite.plugins.io.checkpoint_plugin import CheckpointIO
-from lightning_lite.plugins.io.torch_plugin import TorchCheckpointIO
-from lightning_lite.plugins.io.xla_plugin import XLACheckpointIO
+from lightning_lite.plugins.io.checkpoint_io import CheckpointIO
+from lightning_lite.plugins.io.torch_checkpoint_io import TorchCheckpointIO
```

Comment:
We could do `torch_` the same way we do for `logging_` in trainer. This wouldn't affect users really, because they are supposed to import from the higher plugin level.

Reply:
also we could reduce `_io` as it is the module just before

Reply:
I agree with @awaelchli's proposal.